tools/autograd/gen_variable_type.py

```diff
@@ -201,10 +201,7 @@
 );
 """)
 
-UNPACK_TENSOR = CodeTemplate("""\
-auto${ref} ${arg_name}_ = unpack${suffix}(${arg_name}, "${arg_name}", ${arg_pos});""")
-
-UNPACK_OPTIONS = CodeTemplate("""\
+PACK_TENSOR_OPTIONS = CodeTemplate("""\
```

Comment:
Why pack? "unpack" is used pretty universally in these scripts to mean "process a param and assign the result to a local", which I think is still what this is doing, right?

Reply:
that's how "unpack" was used previously but except for tensor options, that is now gone - so afaik, TensorOptions is the only arguments for which we use that nomenclature now. And to "unpack" TensorOptions in my mind means to take a TensorOptions object and scatter it into ScalarType, Layout, Device.

Reply:
Hmmm what about all this? https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_python_functions.py#L216